app/src/main/java/ro/code4/deurgenta/ui/splashscreen/SplashScreenViewModel.kt

```diff
@@ -13,6 +11,10 @@ class SplashScreenViewModel(sharedPreferences: SharedPreferences) : BaseViewMode
     val loginStatus: LiveData<LoginStatus> = _loginStatus
 
     init {
+        // TODO: currently, there's no way to refresh the token
+        // Uncomment the following line to force the user to sign in again
+//        sharedPreferences.deleteToken()
+
```

Comment:
I've added this line to help us log out and log back in more easily, until we get a way to automatically refresh the auth token.